pokedex_async_redux/lib/features/pokemon_overview_page.dart

```diff
@@ -10,25 +11,50 @@ class PokemonOverviewPage extends StatelessWidget {
     Key? key,
   }) : super(key: key);
 
-  final List<Pokemon> pokemons;
+  final Async<List<Pokemon>> pokemons;
+
+  void _showErrorMessageSnackbar(BuildContext context, String message) {
+    ScaffoldMessenger.of(context).showSnackBar(
+      SnackBar(
+        content: Text(message),
+      ),
+    );
+  }
 
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
       debugShowCheckedModeBanner: false,
-      home: Scaffold(
-        appBar: AppBar(
-          title: const Text(appBarTitle),
-          backgroundColor: appDefaultColor,
-        ),
-        body: Center(
-          child: GridView.builder(
-            gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(crossAxisCount: 2),
-            itemCount: pokemons.length,
-            itemBuilder: (context, index) {
-              final pokemon = pokemons[index];
-              return PokemonCard(pokemon: pokemon);
+      home: Builder(
```

Comment:
why wrap this in a Builder? seems no need for this, u can maximize using the build method's provided context 🙂